### ISSUE TITLE
Parameterize "RequestLimitExceeded" jitter magnitude

### ISSUE THREAD

**Issue by lvangool** [fog/fog-aws#448]
We have a use case where we need greater diversity of jitter (in the case of RequestLimitExceeded). 
The use case is essentially management of multiple AWS resources on behalf of many customers / we're seeing that the jitter differential doesn't handle the burst requests (taking a long time to resolve)

It is currently configured as:
```
jitter = rand(100)
wait_time = ((2.0 ** (1.0 + retries) * 100) + jitter) / 1000.0
```
I'd like to configure something like (pseudo)
```
jitter = rand(100)
jitter_magnitude = @config[:jitter_magnitude] || 1
jitter_adjusted = (jitter / 1000.0) * jitter_magnitude
wait_time = ((2.0 ** (1.0 + retries) * 100) / 1000.0) + jitter_adjusted
```
Any objections?

**Comment by lvangool** [comment]
Add PR: https://github.com/fog/fog-aws/pull/451 
Thanks!

**Comment by geemus** [comment]
closed by #451
